2022-01-15 15:33:24,626 INFO    Thread-13 :23577 [wandb_setup.py:_flush():71] setting env: {'project': 'Amazon-Reviews', 'entity': 'ewebid', 'root_dir': '/Users/ewelina/Desktop/MLOps_Amazon_review_project/src/models', 'run_id': 'xgi9bpgf', 'sweep_param_path': '/Users/ewelina/Desktop/MLOps_Amazon_review_project/src/models/wandb/sweep-j2xo5oqt/config-xgi9bpgf.yaml', 'sweep_id': 'j2xo5oqt'}
2022-01-15 15:33:24,627 INFO    Thread-13 :23577 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-15 15:33:24,627 INFO    Thread-13 :23577 [wandb_init.py:_log_setup():371] Logging user logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/src/models/wandb/run-20220115_153324-xgi9bpgf/logs/debug.log
2022-01-15 15:33:24,627 INFO    Thread-13 :23577 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/src/models/wandb/run-20220115_153324-xgi9bpgf/logs/debug-internal.log
2022-01-15 15:33:24,627 INFO    Thread-13 :23577 [wandb_init.py:init():404] calling init triggers
2022-01-15 15:33:24,627 INFO    Thread-13 :23577 [wandb_init.py:init():409] wandb.init called with sweep_config: {'batch_size': 200, 'drop_out': 0.35, 'epochs': 5, 'lr': 0.001, 'optimizer': 'adam'}
config: {}
2022-01-15 15:33:24,627 INFO    Thread-13 :23577 [wandb_init.py:init():460] starting backend
2022-01-15 15:33:24,628 INFO    Thread-13 :23577 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-15 15:33:24,631 INFO    Thread-13 :23577 [backend.py:ensure_launched():216] starting backend process...
2022-01-15 15:33:24,636 INFO    Thread-13 :23577 [backend.py:ensure_launched():221] started backend process with pid: 23687
2022-01-15 15:33:24,641 INFO    Thread-13 :23577 [wandb_init.py:init():469] backend started and connected
2022-01-15 15:33:24,643 INFO    Thread-13 :23577 [wandb_run.py:_config_callback():956] config_cb None None {'batch_size': 200, 'drop_out': 0.35, 'epochs': 5, 'lr': 0.001, 'optimizer': 'adam'}
2022-01-15 15:33:24,646 INFO    Thread-13 :23577 [wandb_init.py:init():533] updated telemetry
2022-01-15 15:33:24,674 INFO    Thread-13 :23577 [wandb_init.py:init():563] communicating current version
2022-01-15 15:33:27,792 INFO    Thread-13 :23577 [wandb_init.py:init():568] got version response 
2022-01-15 15:33:27,792 INFO    Thread-13 :23577 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-15 15:33:28,788 INFO    Thread-13 :23577 [wandb_init.py:init():606] starting run threads in backend
2022-01-15 15:33:33,800 INFO    Thread-13 :23577 [wandb_run.py:_console_start():1810] atexit reg
2022-01-15 15:33:33,800 INFO    Thread-13 :23577 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-15 15:33:33,801 INFO    Thread-13 :23577 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-15 15:33:33,801 ERROR   Thread-13 :23577 [wandb_run.py:_redirect():1748] Failed to redirect.
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1741, in _redirect
    out_redir.install()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/opt/anaconda3/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-01-15 15:33:33,801 INFO    Thread-13 :23577 [wandb_init.py:init():633] run started, returning control to user process
2022-01-15 15:33:39,494 INFO    Thread-13 :23577 [wandb_run.py:finish():1457] finishing run ewebid/Amazon-Reviews/xgi9bpgf
2022-01-15 15:33:39,494 INFO    Thread-13 :23577 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-15 15:33:39,497 INFO    Thread-13 :23577 [wandb_run.py:_restore():1752] restore
2022-01-15 15:33:39,503 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 757
}

2022-01-15 15:33:40,125 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 757
}

2022-01-15 15:33:40,851 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 6903
}

2022-01-15 15:33:40,961 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18058
}

2022-01-15 15:33:41,066 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18058
}

2022-01-15 15:33:41,173 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18058
}

2022-01-15 15:33:41,277 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18058
}

2022-01-15 15:33:41,381 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18058
}

2022-01-15 15:33:41,488 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18058
  total_bytes: 18058
}

2022-01-15 15:33:41,600 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18058
  total_bytes: 18058
}

2022-01-15 15:33:41,711 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18058
  total_bytes: 18058
}

2022-01-15 15:33:41,841 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18058
  total_bytes: 18058
}

2022-01-15 15:33:41,952 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18058
  total_bytes: 18058
}

2022-01-15 15:33:42,467 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18058
  total_bytes: 18058
}

2022-01-15 15:33:43,397 INFO    Thread-13 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18058
  total_bytes: 18058
}
local_info {
}

2022-01-15 15:33:44,598 INFO    Thread-13 :23577 [wandb_run.py:_append_files():2180] logging synced files
